examples/crypto/crypto-trading-basic.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/crypto-trading-basic.ipynb)"
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/crypto/crypto-trading-basic.ipynb)"
```

Comment:
Oh good catch! Could you please also update `stocks-trading-basic.ipynb` / `options-trading-basic.ipynb` / `options-trading-mleg.ipynb`?